### PR TITLE
[#609] request for manager (connects #609)

### PR DESCRIPTION
**Pull request by razu9861**

### CODE REVIEW

File: init_docs/Text.json [hunk 1]
```diff
@@ -383,7 +383,7 @@
    "Not_Set": "Not Set",
    "Course_Information": "Course Information",
    "level_Single":"Level",
-   "Hi":"Hi",
+   "Hi":"Hello",
```
Comment (dogi) [inline init_docs/Text.json]: hmm - this gets changed but then never used the whole PR-code

please change it back

cause it looks like in line 840 you created `Hello`

**Review by dogi — CHANGES_REQUESTED** [review]
like in last weeks PR https://github.com/open-learning-exchange/BeLL-Apps/pull/615#discussion_r99653384 we have 3 more languages files which should get also treated the same way as `English.txt`

File: app/MyApp/app/views/MemberForm.js [hunk 2]
```diff
@@ -137,6 +137,7 @@ $(function() {
             var that = this
             var currentdate = new Date();
             memberList.each(function (m) {
+                console.log(m)
```
Comment (dogi) [inline app/MyApp/app/views/MemberForm.js]: wonder about this line ;)

**Review by dogi — APPROVED** [review]

**Review by Pr0chin — APPROVED** [review]

**Review by Rupesh87 — APPROVED** [review]

**Review by lmmrssa — APPROVED** [review]

File: app/MyApp/app/views/MemberForm.js (hunk 2 above)
Comment (leomaxi) [reply app/MyApp/app/views/MemberForm.js]: Is this for debugging purposes?

**Review by leomaxi — APPROVED** [review]

**Review by Pr0chin — APPROVED** [review]

**Pull request closed without merging**